backend/.gitignore

```diff
@@ -0,0 +1,329 @@
+<<<<<<< HEAD
+# -------- Ignoring the files and directories that are not required to be pushed to the repository --------
+
+
+
+# For Node.js projects, the following files and directories are not required to be pushed to the repository
+=======
+>>>>>>> 998fe05 (removed npm added yarn functionality)
```

Comment:
Please resolve the merge conflicts here, I don't think you need a `.gitignore` inside the backend folder, as it is already present in the root directory.

Reply:
Sure Sir